configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
Do you recall the reason for bundling readline in the first place? Surely it's one of the most widely deployed packages in any linux -- isn't it like a dependency of the shell?